config/config.yaml

```diff
@@ -66,24 +66,17 @@ assets:
     pdf-style:
         basePath: '%webroot%/public/themes/'
         link: style_pdf.css
-    patientportal-base:
-        basePath: '%webroot%/public/themes/'
-        link: patientportal-base.css
-        rtl:
-            basePath: '%webroot%/public/themes/'
-            link: rtl_patientportal-base.css
+    portal-theme:
+      alreadyBuilt: true
+      link: '%portal_css_header%'
+      autoload: false
+      allowNoLoad: true
```

Comment:
can remove above 2 lines (autoload defaults to false and allowNoLoad is only pertinent when autoload is true)